all-the-icons.el

```diff
@@ -124,140 +124,286 @@
+    ("postcss"      all-the-icons-fileicon "postcss"          :face all-the-icons-dred)
+    ("sss"          all-the-icons-fileicon "postcss"          :face all-the-icons-dred)
+    ("styl"         all-the-icons-alltheicon "stylus"         :face all-the-icons-lgreen)
+    ("csv"          all-the-icons-octicon "graph"             :v-adjust 0.0 :face all-the-icons-dblue)
+    ;; haskell
+    ("hs"           all-the-icons-alltheicon "haskell"        :height 1.0  :face all-the-icons-red)
+    ("chs"          all-the-icons-alltheicon "haskell"        :height 1.0  :face all-the-icons-red)
+    ("lhs"          all-the-icons-alltheicon "haskell"        :height 1.0  :face all-the-icons-red)
+    ("hsc"          all-the-icons-alltheicon "haskell"        :height 1.0  :face all-the-icons-red)
+    ;; Web modes
+    ("inky-haml"    all-the-icons-fileicon "haml"             :face all-the-icons-lyellow)
+    ("haml"         all-the-icons-fileicon "haml"             :face all-the-icons-lyellow)
+    ("htm"          all-the-icons-alltheicon "html5"          :face all-the-icons-orange)
+    ("html"         all-the-icons-alltheicon "html5"          :face all-the-icons-orange)
+    ("inky-er"      all-the-icons-alltheicon "html5"          :face all-the-icons-lred)
+    ("inky-erb"     all-the-icons-alltheicon "html5"          :face all-the-icons-lred)
+    ("erb"          all-the-icons-alltheicon "html5"          :face all-the-icons-lred)
+    ("hbs"          all-the-icons-fileicon "moustache"        :face all-the-icons-green)
+    ("inky-slim"    all-the-icons-octicon "dashboard"         :v-adjust 0.0 :face all-the-icons-yellow)
+    ("slim"         all-the-icons-octicon "dashboard"         :v-adjust 0.0 :face all-the-icons-yellow)
+    ("jade"         all-the-icons-fileicon "jade"             :face all-the-icons-red)
+    ("pug"          all-the-icons-fileicon "pug-alt"          :face all-the-icons-red)
+    ;; Javascript
+    ("d3js"         all-the-icons-alltheicon "d3"             :height 0.8  :face all-the-icons-lgreen)
+    ("re"           all-the-icons-fileicon "reason"           :height 1.0  :face all-the-icons-red-alt)
+    ("rei"          all-the-icons-fileicon "reason"           :height 1.0  :face all-the-icons-dred)
+    ("ml"           all-the-icons-fileicon "ocaml"            :height 1.0  :face all-the-icons-lpink)
+    ("mli"          all-the-icons-fileicon "ocaml"            :height 1.0  :face all-the-icons-dpink)
+    ("react"        all-the-icons-alltheicon "react"          :height 1.1  :face all-the-icons-lblue)
+    ("ts"           all-the-icons-fileicon "typescript"       :height 1.0 :v-adjust -0.1 :face all-the-icons-blue-alt)
+    ("js"           all-the-icons-alltheicon "javascript"     :height 1.0 :v-adjust 0.0 :face all-the-icons-yellow)
+    ("es[0-9]"      all-the-icons-alltheicon "javascript"     :height 1.0 :v-adjust 0.0 :face all-the-icons-yellow)
```

Comment:
This is still regex? I think this can probably be removed. There's no code on Github with an extension that matches this.

Reply:
I probably missed this one, sorry. I'll change it to `"es"`.